packages/modules/localstack/src/localstack-container.test.ts

```diff
@@ -69,4 +69,18 @@ describe("LocalStackContainer", () => {
     await container.stop();
     await awsCliInDockerNetwork.stop();
   });
+
+  it("should use access environment variables and default creds", async () => {
+    const container = await new LocalstackContainer()
+      .withEnvironment({
+        DEFAULT_REGION: "eu-west-3",
```

Comment:
Where does the `DEFAULT_REGION` convention come from? I had a quick look at the docs and found https://docs.localstack.cloud/references/configuration/#legacy, which says that it's no longer supported.

Reply:
Yes, I think you're right. Many things have changed with 3.x and I should have refreshed my knowledge. The point is the default image is still `localstack/localstack:2.2.0`. As multi-region has been introduced incrementally, this may not affect the module, but it may be nice to upgrade to 3.x to definitely get rid of this question. What do you think?